<launch>
    <node name="keypoints_preprocessing" pkg="pamaral_object_grasping_pattern_recognition" type="keypoints_preprocessing.py" output="screen"/>
</launch>